unblob/strategies.py

```diff
@@ -55,9 +55,9 @@ def search_chunks_by_priority(  # noqa: C901
                 log.info("Found valid chunk")
                 all_chunks.append(chunk)
 
-                if chunk.size == file_size:
-                    log.info("This Chunk represents the whole file")
-                    return [chunk]
+                if sum([chunk.size for chunk in all_chunks]) == file_size:
```

Comment:
This is not enough, the chunks has to be aligned.
Write a new function so you can unit test this.

Reply:
Yes, thank you.